scss/components/notification.scss

```diff
@@ -0,0 +1,312 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../functions";
+
+$block: #{$fd-namespace}-notification;
+.#{$block} {
+    // variables
+    $fd-notification-max-width: 60rem !default;
+    $fd-notification-max-width-m: 640px !default;
+    $fd-notification-min-width-m: 320px !default;
+    $fd-notification-width-s: 320px !default;
+
+    $fd-notification-background-color: var(--sapList_Background, $sapList_Background) !default;
+    $fd-notification-darken-background-color: $sapList_Background !default;
+
+    $fd-notification-box-shadow: var(--sapContent_Shadow0, $sapContent_Shadow0) !default;
+
+    $fd-notification-border-radius: 0.25rem !default;
+    $fd-notification-group-border-radius-first-child: 0.25rem 0.25rem 0 0 !default;
+    $fd-notification-group-border-radius-last-child: 0 0 0.25rem 0.25rem !default;
+
+    $fd-notification-indicator-color-error: var(--sapErrorColor, $sapErrorColor) !default;
+    $fd-notification-indicator-color-warning: var(--sapWarningColor, $sapWarningColor) !default;
+    $fd-notification-indicator-color-success: var(--sapSuccessColor, $sapSuccessColor) !default;
+    $fd-notification-indicator-color-information: var(--sapInformationColor, $sapInformationColor) !default;
+
+    $fd-notification-indicator-border-color-error: var(--sapErrorBorderColor, $sapErrorBorderColor) !default;
+    $fd-notification-indicator-border-color-warning: var(--sapWarningBorderColor, $sapWarningBorderColor) !default;
+    $fd-notification-indicator-border-color-success: var(--sapSuccessBorderColor, $sapSuccessBorderColor) !default;
+    $fd-notification-indicator-border-color-information: var(--sapInformationBorderColor, $sapInformationBorderColor) !default;
+
+    $fd-notification-title-color: var(--sapGroup_TitleTextColor, $sapGroup_TitleTextColor) !default;
+    $fd-notification-content-text-color: var(--sapContent_LabelColor, $sapContent_LabelColor) !default;
```

Comment:
so I think if we setup postcss correctly, none of this is necessary. You would just need `var(--sapContent_LabelColor)` and that's it.